src/js/components/Tab/Tab.js

```diff
@@ -70,6 +73,8 @@ const Tab = forwardRef(
           theme.tab.border.color || theme.global.control.border.color;
         if (active) {
           borderColor = theme.tab.border.active.color || borderColor;
+        } else if (disabled && theme.tab.border.disabled) {
+          borderColor = theme.tab.border.disabled.color || borderColor;
```

Comment:
Going over the flow of resolving the border color, if somehow the tab is defined as `active` while `disabled`, the `disabled` will be ignored -- so `active` will triumph. I wanted to hear your thoughts on maybe put a warning message for the combined usage of both ` active` & `disabled`, and I'm thinking maybe `plain` & `disabled` as well. let me know if you think it will be useful.

Reply:
I think implementing warning for `active && disabled` would beneficial. I will add.

For `plain && disabled`, I think this is a valid combination. I could see a use case where I have a `plain` Tab for my style reasons, and would want to apply `disabled` in addition to `plain`.

Reply:
Added warning for the `active && disabled` case as they are conflicting states.

Opted not to add warning for the `plain && disabled` as having a `disabled` state on `plain` styled Tab is reasonable scenario.

Reply:
Thanks! I believe the users will find it helpful.

Regarding `plain & disabled`, I noticed that on the changes you've made in `StyledTab.js`, the `disabled` style will only apply on a button that isn't `plain`, so I might be missing something but from the code on this PR I don't see a way for `disabled` style to be applied on a `plain` button. Is that the intention?

Reply:
That was the intention originally, but based on the above, I have come around and believe that applying disabled state to a plain style is reasonable. It would also be consistent with the behavior for `<Button label="Button" plain disabled />`. Good catch. I will modify.

Reply:
Great, thanks.

Reply:
FYI, confirming that the `plain` restriction on StyledTab was removed.